docs/jwst/outlier_detection/outlier_detection.rst

```diff
@@ -58,7 +58,7 @@ Specifically, this routine performs the following operations:
      should be used when resampling to create the output mosaic.  Any pixel with a
      DQ value not included in this value (or list of values) will be ignored when
      resampling.
-   * Resampled images will be written out to disk as `_outlier_i2d.fits` by default.
+   * Resampled images will be written out to disk as `_<asn_id>_outlier_i2d.fits` by default.
```

Comment:
Resampled images will be written out to disk with the suffix '_<asn_id>_outlier_i2d.fits' if the input model container has an <asn_id>, otherwise the suffix will be '_outlier_i2d.fits' by default.

Reply:
Thanks! I updated the text with your suggestion: 509375f68909e310b6c1c389c36c81c51f2851eb